### DISCUSSION TITLE
How to set loading policies for bucketized resources

### DISCUSSION THREAD

**Discussion by djdkochasoft** [frePPLe/frepple#505] [Q&A]
Hi there!

In frepple documentation on resources [here](https://frepple.org/docs/6.20/model-reference/operation-resources.php?highlight=loading%20policies), it says that:
"On bucketized resources the capacity is consumed from the capacity bucket at a single moment of time. Different loading policies can be specified:"

### **Where can we specify these loading policies?**

Policies mentioned in the documentation are:
    default:
    load_bucketized_end:
    load_bucketized_start:
    load_bucketized_percentage:

I cannot find this configuration in the resources / operation-resources screens

Your help is much appreciated. Thanks!

**Comment by jdetaeye** [comment]
The planning engine has this capability, but it is not configurable in the user interface.   

In practice, we always use load_bucketized_start.
The two other policies are relatively untested. If you want to use those, extra unit tests are needed.
